bin/perseo

```diff
@@ -63,6 +63,7 @@ function loadConfiguration() {
     }
     if (process.env.PERSEO_MONGO_HOST) {
         config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
+        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
```

Comment:
I think this comment was finally taken into account in a next PR: https://github.com/telefonicaid/perseo-fe/pull/108